### PR TITLE
(feat) Update procedure form to include body site

### PR DESCRIPTION
**Pull request by agesa3**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Screenshots

Screenshot 2024-04-16 at 14 42 50

### CODE REVIEW

**Review by jecihjoy — APPROVED** [review]
Hi @agesa3 was this  requested by the BA, if yes we can try to make it a selectable list which can pick of a `concept set`

**Comment by jecihjoy** [comment]
I will defer this to @ojwanganto to help advice whether we need three different sections (body site, additional instructions and comments to fulfiller) for captuire procedure ordering information

File: src/form/procedures-orders/add-procedures-order/procedures-order-form.component.tsx [hunk 1]
```diff
@@ -92,6 +92,10 @@ export function ProceduresOrderForm({
     ) || {}
   ).required;
 
+  const {
+    items: { answers: bodySiteItems },
+  } = useConceptById("162668AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA");
```
Comment (ojwanganto) [inline src/form/procedures-orders/add-procedures-order/procedures-order-form.component.tsx:97]: can we use a config for this?

**Review by ojwanganto — CHANGES_REQUESTED** [review]

**Comment by ojwanganto** [comment]
@donaldkibet - please re-review this.

**Review by donaldkibet — APPROVED** [review]
Thanks @agesa3 LGTM

**Comment by ojwanganto** [comment]
So my thoughts are:
**body site:** is the target body part for the procedure, 
**additional instructions:**  this is instructions about the procedure and 
**comments to fulfiller:** is anything you would want to communicate to the person handling the procedure

@jecihjoy - do you see conflicts?

**Comment by jecihjoy** [comment]
> So my thoughts are: **body site:** is the target body part for the procedure, **additional instructions:** this is instructions about the procedure and **comments to fulfiller:** is anything you would want to communicate to the person handling the procedure
> 
> @jecihjoy - do you see conflicts?

Makes sense, we can proceed with the changes

**Pull request merged**